packages/common/src/authorization/organizationMemberAbility.test.ts

```diff
@@ -761,7 +811,18 @@ describe('Organization member permissions', () => {
         it('can use the SemanticViewer', () => {
             expect(ability.can('manage', 'SemanticViewer')).toEqual(true);
         });
+
+        it('cannot create a project in organization they belong to', () => {
```

Comment:
[non blocker] Are we missing the test were they can create a preview in the same org?

Reply:
improved specs including this one